extensions/amp-story/0.1/amp-story.css

```diff
@@ -926,3 +926,126 @@ amp-story[standalone].i-amphtml-story-landscape .i-amphtml-story-no-rotation-ove
 .i-amphtml-story-button-container {
   display: none !important;
 }
+
+/* Default loading spinner */
```

Comment:
All styles in this file should be `!important` (I should investigate making conformance tests...)

If you want it to be user-overridable, you can add it to `amp-story-user-overridable.css`.  I don't think that's the case here.

Reply:
Done